src/main/host/syscall/format.rs

```diff
@@ -77,12 +119,15 @@ macro_rules! simple_pointer_impl {
             fn fmt(
                 &self,
                 f: &mut std::fmt::Formatter<'_>,
+                options: FmtOptions,
                 mem: &MemoryManager,
             ) -> std::fmt::Result {
-                match mem.memory_ref(TypedPluginPtr::new::<$type>(self.ptr, 1)) {
-                    Ok(vals) => write!(f, "{} ({:p})", &(*vals)[0], self.ptr),
+                match (options, mem.memory_ref(TypedPluginPtr::new::<$type>(self.ptr, 1))) {
+                    (FmtOptions::Standard, Ok(vals)) => write!(f, "{} ({:p})", &(*vals)[0], self.ptr),
+                    (FmtOptions::Deterministic, Ok(_vals)) => write!(f, "<pointer>"),
```

Comment:
Do we still want the unused `_vals` here?

Reply:
Changed to just `_`.